save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/validation/Validatable.kt

```diff
@@ -0,0 +1,16 @@
+/**
+ * Module that defines an interface for objects validation.
+ */
+
+package com.saveourtool.save.validation
+
+/**
+ * [Validatable] should be implemented by all the classes where we need validation
+ */
+interface Validatable {
+    /**
+     * @return true if object is valid, false otherwise
+     */
+    @Suppress("FUNCTION_BOOLEAN_PREFIX")
+    fun validate(): Boolean = true
```

Comment:
On a second thought, this is quite poor API in terms of expressiveness. Consider `java.io` API: method `File.delete` returns `false` if file has not been deleted. Now consider `java.nio`: `Files.delete` throws an exception in case deletion was not successful with detailed reasons.

Reply:
Pls take a look at https://github.com/jquense/yup. Will this help us on the frontend?